### PR TITLE
adds setup/loop

### PR DESCRIPTION
**Pull request by tomaszduda23**
I also changed way of patching libs since previous one was silent on error.

https://github.com/pikvm/kvmd/pull/90

### CODE REVIEW

**Comment by tomaszduda23** [comment]
It does not build in vscode

**Comment by tomaszduda23** [comment]
It should be okay now

**Comment by tomaszduda23** [comment]
https://github.com/pikvm/kvmd/pull/90

File: hid/patches/hid-no-singletones.patch [hunk 1]
```diff
@@ -45,7 +45,7 @@ diff -u -r a/src/SingleReport/BootMouse.h b/src/SingleReport/BootMouse.h
 diff -u -r a/src/SingleReport/SingleAbsoluteMouse.cpp b/src/SingleReport/SingleAbsoluteMouse.cpp
 --- a/src/SingleReport/SingleAbsoluteMouse.cpp	2020-11-17 18:39:35.314843889 +0300
 +++ b/src/SingleReport/SingleAbsoluteMouse.cpp	2020-11-17 18:59:12.189345326 +0300
-@@ -139,6 +139,6 @@
+@@ -144,6 +144,6 @@
```
Comment (mdevaev) [inline hid/patches/hid-no-singletones.patch:48]: revert it please, it's not necessary

**Pull request merged**